app/models/value.rb

```diff
@@ -69,13 +71,24 @@ def as_txt(values, version:, locale: "en")
 
     private
 
+    def row_from_data_with_position(data, base_attributes_by_friendly_id)
+      row = row_from_data(data)
+
+      matching_attribute = base_attributes_by_friendly_id[row["primary_attribute_friendly_id"]]
+
+      return row unless matching_attribute["sorting"] == "custom"
```

Comment:
Nit: Does this need to be an enum? If we don't expect to add other sort types perhaps `matching_attribute["custom_sort"]` returns a boolean.

Reply:
I don't really want to prescribe how we're going to use it quite yet. If it evolves then this is an easy fix